README.md

```diff
@@ -26,6 +26,12 @@ Product documentation for StackStorm is maintained in this repository. These doc
 #### Build locally on Linux
 Follows these steps to build the docs locally:
 
+Install the dependencies:
+
+For Debian/Ubuntu: ``sudo apt-get install libpython-dev libssl-dev``
+
+For RHEL/CentOS: `` sudo yum install libpython-dev libssl-dev``
```

Comment:
Package names are different on RHEL/CentOS. You'll need to check, but I think it's python-devel and openssl-devel.